.eslintrc

```diff
@@ -0,0 +1,12 @@
+{
+	"extends": "humanmade",
+	"globals": {
+		"Altis": "readonly",
+		"wp": "readonly",
+		"moment": "readonly"
+	},
+	"rules": {
+		"no-multi-str": "off",
+		"no-console": "off"
```

Comment:
I think this wants to be on for non-local dev?

Reply:
No I have an error message in the Test js class to warn if it's been implemented incorrectly.

Reply:
A-hoy